### PR TITLE
Add currency name COP

### PR DESCRIPTION
**Pull request by Stenfrank**
Currency name COP

### CODE REVIEW

File: src/Legacy/Numbers/Words/Locale/Es.php [hunk 1]
```diff
@@ -94,7 +94,8 @@ class Es extends Words
         'XAF' => [['franco CFA', 'francos CFA'], ['céntimo']],
         'XOF' => [['franco CFA', 'francos CFA'], ['céntimo']],
         'YUM' => [['dinar', 'dinares'], ['para']],
-        'ZAR' => [['rand'], ['cent']]
+        'ZAR' => [['rand'], ['cent']],
+        'COP' => [['peso', 'pesos'], ['centavo', 'centavos']]
```
Comment (kwn) [inline src/Legacy/Numbers/Words/Locale/Es.php:98]: Could you please keep the alphabetical order?

**Review by kwn — CHANGES_REQUESTED** [review]
1 minor update requested. Could you please add a few unit tests?

File: src/Legacy/Numbers/Words/Locale/Es.php (hunk 1 above)
Comment (Stenfrank) [reply src/Legacy/Numbers/Words/Locale/Es.php]: Alphabetical order applied

**Comment by Stenfrank** [comment]
> 1 minor update requested. Could you please add a few unit tests?

Aggregate tests

**Review by kwn — APPROVED** [review]

**Comment by kwn** [comment]
Thanks!!

**Pull request merged**